listenbrainz/labs_api/labs/api/artist_credit_from_artist_mbid.py

```diff
@@ -23,6 +23,9 @@ def outputs(self):
     def fetch(self, params, count=-1, offset=-1):
 
         with psycopg2.connect(config.MB_DATABASE_URI) as conn:
+            if not config.MB_DATABASE_URI:
```

Comment:
I had earlier looked from mobile so missed this. This needs to be outside the with,